app/models/space_host_policy.rb

```diff
@@ -0,0 +1,60 @@
+require 'conjur_client'
+
+class SpaceHostPolicy
+  include ConjurApiModel
+
+  class << self
+    def create(org_id, space_id)
+      SpaceHostPolicy.new(org_id, space_id).create
+    end
+  end
+
+  def initialize(org_id, space_id)
+    @org_id = org_id
+    @space_id = space_id
+  end
+
+  def create
+    result = conjur_api.load_policy(space_policy, template_create, method: Conjur::API::POLICY_METHOD_POST)
+    
+    created_role = result.created_roles.values.first
+
+    if created_role != nil
+      variable_id = "#{space_policy}/#{space_host_api_key_variable_id}"
+      set_variable(variable_id, created_role['api_key'])
+    end
+  end
+
+  private
+
+  def template_create
+    <<~YAML.strip + "\n"
+    - !host
+      id: #{space_host_id}
```

Comment:
What do you think about leaving off an `id` here and just letting the host take the GUID name, the same as the Space policy and layer?

Reply:
Okay so it would be named the same as the space policy? That makes sense sure.